pages/sponsors.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import Image from "next/image";
+import styles from "@/styles/Sponsors.module.scss";
+
+const laCtfLogos = [
+  { src: "/images/crowdstrike.png", alt: "CrowdStrike" },
```

Comment:
These files don't exist yet - they need to be added to the /public/images folder